Cliqz/Settings/CliqzAppSettingsOptions.swift

```diff
@@ -106,13 +106,22 @@ class ComplementarySearchSetting: Setting, SearchEnginePickerDelegate {
     }
     
     override func onClick(_ navigationController: UINavigationController?) {
+        #if PAID
+        let searchEnginePicker = LumenSearchEnginePicker()
+        searchEnginePicker.profile = self.profile
+        searchEnginePicker.searchEnginesUpdated = { [weak self] in
+            searchEnginePicker.engines = self?.models
+            searchEnginePicker.tableView.reloadData()
+        }
+        #else
         let searchEnginePicker = SearchEnginePicker()
+        #endif
         // Order alphabetically, so that picker is always consistently ordered.
         // Every engine is a valid choice for the default engine, even the current default engine.
-        let searchEngines = profile.searchEngines
-        searchEnginePicker.engines = searchEngines.orderedEngines.sorted { e, f in e.shortName < f.shortName }
+        searchEnginePicker.engines = self.models
```

Comment:
You moved the sorting code into a function -- please also move the comment about sorting into the function :)